noir-projects/aztec-nr/aztec/src/prelude.nr

```diff
@@ -6,7 +6,6 @@ pub use crate::{
         note_interface::{NoteInterface, NullifiableNote},
         note_viewer_options::NoteViewerOptions,
         retrieved_note::RetrievedNote,
-        utils::compute_note_hash_and_optionally_a_nullifier as utils_compute_note_hash_and_optionally_a_nullifier,
```

Comment:
This re-export was unused and I see no reason for having it. I think it was just forgottent.